.swiftlint.yml

```diff
@@ -7,10 +7,10 @@ disabled_rules:
   - cyclomatic_complexity
   - opening_brace
   - todo
-
-line_length: 120
-
-function_body_length: 50
+  - function_parameter_count
+  - line_length
+  - function_body_length
+  - file_length
```

Comment:
Any specific need for this change? I think we had pretty reasonable limits here and it doesn't look like this PR is exceeding them.

Reply:
SwiftLint action somehow reports errors https://github.com/swiftwasm/carton/actions/runs/2397807140 for the whole code base not only for the diff.

Reply:
Ah, that's fair. Is it a required check? If so, let's make SwiftLint not required and maybe revert these config changes and we'll cleanup warnings in a separate PR?

Reply:
ok